src/main/res/values/strings.xml

```diff
@@ -1283,8 +1283,8 @@
 	<string name="image_gallery_no_image_present_message">The specified image could not be found. It may have been deleted.</string>
 
 	<!-- 2020-09-04 -->
-	<string name="reddit_login_browser_popup_line_1">To login, you will now be taken to Reddit in a web browser.</string>
-	<string name="reddit_login_browser_popup_line_2">When the login is complete, you should be redirected back to RedReader. If the login fails, you can try a different browser using the checkbox below.</string>
+	<string name="reddit_login_browser_popup_line_1">To log in, you will now be taken to Reddit in a web-browser.</string>
+	<string name="reddit_login_browser_popup_line_2">After logging in, you should be redirected back to RedReader. Try a different browser if you can\'t log in by using the checkbox below.</string>
```

Comment:
Preferred the original (although "After logging in" is fine)

Reply:
Original is a needless arbitration to what the user is doing. It also isn't different from the user doing it, as when done, the login is also complete. Fixed to "web-browser"

Reply:
The new version is a garden path sentence -- it initially reads as "(Try a different browser) if you can't (log in by using the checkbox below). Please revert!

Reply:
Also web browser was originally correct, it's not hyphenated in English!